<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProposalDialog</class>
 <widget class="QDialog" name="ProposalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Budget Proposal</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>11</number>
   </property>
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="styleSheet">
      <string notr="true">margin: 12px;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Please fill out the form below to create a voting request. &lt;/p&gt;&lt;p&gt;Before beginning make sure your wallet is unlocked to pay the fee of &lt;span style=&quot;color:#6ff34d&quot;&gt;5000 ZNN&lt;/span&gt;. After the request is submitted you will be shown the transaction hash field and will need to wait for confirmations on the transaction before being able to submit the request for voting.&lt;/p&gt;&lt;p&gt;The screen will update to help guide you, it is never recommended to close the dialog, it will close once completed automatically.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <property name="horizontalSpacing">
        <number>11</number>
       </property>
       <property name="verticalSpacing">
        <number>11</number>
       </property>
       <property name="leftMargin">
        <number>11</number>
       </property>
       <property name="topMargin">
        <number>11</number>
       </property>
       <property name="rightMargin">
        <number>11</number>
       </property>
       <property name="bottomMargin">
        <number>11</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Title:</string>
         </property>
         <property name="buddy">
          <cstring>nameEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QValidatedLineEdit" name="nameEdit" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>URL:</string>
         </property>
         <property name="buddy">
          <cstring>urlEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QValidatedLineEdit" name="urlEdit" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Block start:</string>
         </property>
         <property name="buddy">
          <cstring>blockStart</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QValidatedLineEdit" name="blockStart" native="true"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Block end:</string>
         </property>
         <property name="buddy">
          <cstring>blockEnd</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QValidatedLineEdit" name="blockEnd" native="true"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="hashLabel">
         <property name="text">
          <string>TXID:</string>
         </property>
         <property name="buddy">
          <cstring>hashEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QValidatedLineEdit" name="hashEdit" native="true"/>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="confirmLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="acceptButton">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QWidget</extends>
   <header>qvalidatedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
